src/ui/public/notify/notify.less

```diff
@@ -8,21 +8,21 @@
     margin: 0;
     padding: 0;
     list-style: none;
+  }
+
+  .toaster-countdown {
+    .badge {
```

Comment:
Looks like this nesting here is to override specificity from Bootstrap. 

<img width="614" alt="screen shot 2016-08-24 at 2 07 26 pm" src="https://cloud.githubusercontent.com/assets/908371/17948108/3e6d0c9c-6a04-11e6-96b5-f61fdcdb8b73.png">

A `.toaster-container .toaster-countdown__badge` selector that gives the white color doesn't override the `.btn-danger .badge` selector that gives the red font color.